specification.md

```diff
@@ -0,0 +1,152 @@
+  - **Date-time**
+- `project-url` (Required): URI for the project repository that this document pertains to.
+  - **String** beginning with **https://**
+- `commit-hash` (Required): The last commit on the repository that this document pertains to.
+  - **String** matching the git SHA format
+- `changelog`: URI for the changelog for the project.
+  - **String** beginning with **https://**
+- `license`: URI for the license for the corresponding version of the project.
+  - **String** beginning with **https://**
+- `parent-security-yaml`: If the document is mirrored from another, give a URI pointing to the original.
+  - **String** beginning with **https://**
+
+## Project Lifecycle
+The project-lifecycle section contains information about how the project is maintained. This is the first of many places where we’ll see conditional requirements in the schema.
+
+```yaml
+project-lifecycle:
+  stage: active
+  roadmap: https://foo.bar/roadmap.html
+  bug-fixes-only: false
+  core-maintainers:
+    - https://github.com/scovetta
+    - joe.bob@email.com
+```
+
+Note that the last value on this list is a conditional requirement, making it optional if the project is no longer being maintained. We’ll see more values like this in the following sections.
+- `stage` (Required): The project’s maintenance stage.
+  - **String**. Values may be active or inactive.
+- `roadmap`: URI to information about the project’s roadmap.
+  - **String** beginning with **https://**
+- `bug-fixes-only`: Is the project active, but only accepting bugfix changes?
+  - **Boolean**
+- `core-maintainers`: Required if bug-fixes-only is true or stage is active. Contact information for the project’s maintainers, such as email or URI.
+  - **List** of **String** entries
+
+## Contribution Policy
+The contribution-policy section describes the decisions made by the project regarding when and how to accept contributions.
+
+```yaml
+contribution-policy:
+  accepts-pull-requests: true
+  accepts-automated-pull-requests: true
+  automated-tools-list:
+    - automated-tool: example/security-research
+      action: denied
+      path:
+        - main/foo/bar
+        - main/examples
+      comment: |
+        Lorem ipsum dolor sit amet, consectetur adipisci elit,
+        sed do eiusmod tempor incidunt ut labore et dolore magna aliqua.
+  contributing-policy: https://example.com/development-policy.html
+  code-of-conduct: https://example.com/code-of-conduct.html
+```
+
+- `accepts-pull-requests` (Required): Does your project accept PRs from non-maintainers?
+  - **Boolean**
+- `accepts-automated-pull-requests` (Required): Does your project accept PRs from bots or automated tools?
+  - **Boolean**
+- `automated-tools-list`: If you have exceptions to accepts-automated-pull-requests, such as allowing select tools or blocking select tools, review the specification for more information about how to list those exceptions here.
```

Comment:
Suggestion:
If you expect to accept (or not accept) pull requested generated by tools, you can specify that policy here. (Note that this is just a *request* and doesn't actually prevent pull requests from being opened.)

Reply:
Updated